v3/integrations/nroci/example/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/newrelic/go-agent/v3/integrations/nroci"
+	"github.com/newrelic/go-agent/v3/newrelic"
+	"github.com/oracle/nosql-go-sdk/nosqldb"
+)
+
+func main() {
+	app, err := newrelic.NewApplication(
+		newrelic.ConfigAppName("Basic NOSQLOCI App"),
+		newrelic.ConfigLicense(os.Getenv("NEW_RELIC_LICENSE_KEY")),
+		newrelic.ConfigDebugLogger(os.Stdout),
+	)
+	if err != nil {
+		panic(err)
+	}
+	app.WaitForConnection(10 * time.Second)
```

Comment:
Just to avoid misunderstandings when people follow our example code with their own implementations, we like to add a comment here saying that `WaitForConnection` is probably only needed for very short-lived programs or ones that have some specific need for the app to wait for instrumentation to be going before getting on with their work.